docs/application/native/guides/multimedia/image-classification.md

```diff
@@ -0,0 +1,211 @@
+        dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+    /* Fill the dataBuffer to g_source */
+    error_code = mv_source_fill_by_buffer(imagedata.g_source, dataBuffer, (unsigned int)bufferSize,
+                                          (unsigned int)width, (unsigned int)height, MEDIA_VISION_COLORSPACE_RGB888);
+    if (error_code != MEDIA_VISION_ERROR_NONE)
+        dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+
+    free(dataBuffer);
+    dataBuffer = NULL;
+    ```
+
+3. To classify the `sample.jpg` image, create a `g_inference` media vision inference handle:
+
+    ``` c
+    error_code = mv_inference_create(&imagedata.g_inference);
+    if (error_code != MEDIA_VISION_ERROR_NONE)
+        dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
+    ```
+
+4. Configure `g_engine_config` with classification model data to classify image. In the following example, TensorFlow Lite model is used and `data.tflite` and `label.txt` are in `<OwnDataPath>`. Model data is available in open model zoo such as [hosted model zoo](https://www.tensorflow.org/lite/guide/hosted_models#floating_point_models):
+
+    ``` c
+    #define MODEL_DATA "OwnDataPath/data.tflite"
+    #define MODEL_LABEL "OwnDataPath/label.txt"
+
+    error_code = mv_engine_config_set_string_attribute(handle,
+                      MV_INFERENCE_MODEL_WEIGHT_FILE_PATH,
+                      MODEL_DATA);
+
+    error_code = mv_engine_config_set_string_attribute(handle,
+                      MV_INFERENCE_MODEL_USER_FILE_PATH,
+                      MODEL_LABEL);
+
+    error_code = mv_engine_config_set_double_attribute(handle,
+                      MV_INFERENCE_MODEL_MEAN_VALUE,
+                      127.5);
+
+    error_code = mv_engine_config_set_double_attribute(handle,
+                      MV_INFERENCE_MODEL_STD_VALUE,
+                      127.5);
+
+    error_code = mv_engine_config_set_double_attribute(handle,
+                      MV_INFERENCE_CONFIDENCE_THRESHOLD,
+                      0.0);
+
+    error_code = mv_engine_config_set_int_attribute(handle,
+                      MV_INFERENCE_BACKEND_TYPE,
+                      MV_INFERENCE_BACKEND_TFLITE);
+
+    error_code = mv_engine_config_set_int_attribute(handle,
+                      MV_INFERENCE_INPUT_TENSOR_WIDTH,
+                      224);
+
+    error_code = mv_engine_config_set_int_attribute(handle,
+                      MV_INFERENCE_INPUT_TENSOR_HEIGHT,
+                      224);
+
+    error_code = mv_engine_config_set_int_attribute(handle,
+                      MV_INFERENCE_INPUT_TENSOR_CHANNELS,
+                      3);
+    ```
+    The details of configuration attributes such as `MV_INFERENCE_MODEL_WEIGHT_FILE_PATH` of Inference API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) applications) can be found.
```

Comment:
How about below sentence? @rishu-samsung, please check the grammar so that @tyoungroy can update this sentence.

For more information on the configuration attributes such as `MV_INFERENCE_MODEL_WEIGHT_FILE_PATH`, see the Media Vision Inference API (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__VISION__INFERENCE__MODULE.html) applications).

Reply:
With @rishu-samsung 's comment, i will update the sentence.

Reply:
@mijin85cho One slight change, remove 'the' before 'Media Vision Inference API'
Suggested edit:
...such as `MV_INFERENCE_MODEL_WEIGHT_FILE_PATH`, see Media Vision Inference API (in mobile and wearable applications).

@tyoungroy Kindly update the sentence and fix the conflict regarding TOC_all file.